src/nodes.coffee

```diff
@@ -2110,7 +2112,7 @@ exports.Param = class Param extends Base
     @reference = node
 
   isComplex: ->
-    @name.isComplex()
+    @name.isComplex() or (@value? and @value instanceof Call)
```

Comment:
Bit of a nit-pick but the check for `@value?` is redundant (`@value instanceof Call` doesn't depend on the existence of `@value`).